app/scripts/views/icon.js

```diff
@@ -0,0 +1,17 @@
+import React, {PropTypes} from 'react'
```

Comment:
Because that's something different. To use jsx you need to have the top level `React` in scope, and for the `PropTypes` I don't want to write `React.PropTypes`. For more details see [this article](http://www.2ality.com/2014/09/es6-modules-final.html)